apps/back/src/Controller/ExpenseController.php

```diff
@@ -58,11 +58,13 @@ public function addExpense(
                     $em->persist($expense);
                     $em->flush();
                     $this->addFlash('success', 'Dépense ajoutée !');
-                    return $this->redirectToRoute('expense_list');
+                    $id = $expense->getGroup()->getId();
+                    return $this->redirectToRoute('expense_list', ['id' => $id]);
             }
         } catch (ExpenseWithoutParticipantsException $e) {
             $this->addFlash('error', 'Veuillez cocher au moins 1 participant à cette dépense.');
         } catch (Throwable $e) {
+            throw $e;
             $this->addFlash('error', 'Erreur dans le formulaire, veuillez saisir tous les champs.');
```

Comment:
If you throw your error before adding the flash notification, you will never have your notification as the `throw` act like a `return`.

Reply:
But, seeing your controller, I don't think you need to throw an execption indeed 😅